### PR TITLE
Add frontend implementation for Insights feature

### PR DESCRIPTION
**Pull request by nghialv**
**What this PR does / why we need it**:

Screen Shot 2022-11-29 at 15 53 23
Screen Shot 2022-11-29 at 15 53 48

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: web/src/components/insight-page/index.tsx [hunk 1]
```diff
@@ -27,6 +27,12 @@ export const InsightIndexPage: FC = memo(function InsightIndexPage() {
       : undefined
   );
 
+  const selectedLabels = useAppSelector<string>(
+    (state) => "{" + state.insight.labels.join(", ") + "}"
+  );
```
Comment (khanhtc1202) [inline web/src/components/insight-page/index.tsx:32]:
```suggestion
const selectedLabels = useAppSelector<string>((state) =>
    state.insight.labels.length !== 0
      ? "{" + state.insight.labels.join(", ") + "}"
      : ""
  );
```
nits

Comment (nghialv) [reply web/src/components/insight-page/index.tsx:32]: 👍

**Review by khanhtc1202 — DISMISSED** [review]
🍠 🍠

**Review by knanao — DISMISSED** [review]
nice, please fix the lint error. 🚀

**Comment by nghialv** [comment]
@khanhtc1202 @knanao Fixed the lint error. Please re-approve this.

**Review by khanhtc1202 — APPROVED** [review]

**Pull request merged**